### PR TITLE
Cosmos fixture for CurrencyInputPanel component

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Example of fixture for CurrencyInputPanel component

image

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
As a follow up, we should add another build target like the explorer:

https://github.com/cowprotocol/explorer/pull/192#issuecomment-1220523334

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Editor configuration, see https://editorconfig.org
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply .editorconfig:1]: It doesn't affect prettier, just tells the IDE how to generate code

[the remark this answers is not in the document]
Comment (alfetopito) [reply .editorconfig:1]: Right, but I mean that prettier also integrates with the IDE and for example applies prettier rules onSave (if configured to do so)

[the remark this answers is not in the document]
Comment (shoom3301) [reply .editorconfig:1]: Oh, I see. Prettier works with formatting of existed code. Editorconfig helps with generating new code.

**Pull request merged**